Changelog.md

```diff
@@ -5,6 +5,9 @@ Changelog
 
 * Use ceil in `ResettingController` for a better token lifetime approximation
 * Removed unused translation keys
+* Removed form deprecations
+* Added `symfony/templating` as a dependency
```

Comment:
I saw that this dependency was introduced mostly to "fix tests", but if the dependencies requirements have changed, I always find this as important because it can introduce potential conflicts with versionning. Even though it is not very likely to happen, I thought it was a good practice to mention the changes in the package's dependencies 😕